core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -284,6 +284,7 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
       masterClient.setAttribute(path, options);
+      LOG.info("Set attributes of file {} to ({})", path.getPath(), generateLogInfo(options));
```

Comment:
SetAttributeOptions#toString method simply transfers unset attributes to "null" string, and I just thought it's not readable.